Sources/DistributedCluster/_ActorShell.swift

```diff
@@ -57,7 +57,7 @@ public final class _ActorShell<Message: Codable>: _ActorContext<Message>, Abstra
     internal let _dispatcher: MessageDispatcher
 
     @usableFromInline
-    internal var _system: ClusterSystem?
+    internal weak var _system: ClusterSystem?
```

Comment:
heh yeah i can see that... previously actor trees destroyed things and now we don't i guess